srcpkgs/impala/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'impala'
+pkgname=impala
+version=0.2.4
+revision=1
+build_style=cargo
+depends="libgcc iwd"
+short_desc="TUI for managing wifi on Linux"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="GPL-3.0-only"
+homepage="https://github.com/pythops/impala"
+changelog="https://raw.githubusercontent.com/pythops/impala/main/Release.md"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```

Comment:
remove the $homepage variable and just put the full path there, makes it easier to copy.